src/lib/file-picker/_core.scss

```diff
@@ -0,0 +1,78 @@
+@use './token-utils' as *;
+@use '../core/styles/typography';
+@use '../core/styles/theme';
+@forward './token-utils';
+
+@mixin host() {
+ display: block;
+ height: 100%;
+ width: 100%;
+}
+
+@mixin base() {
+ display: grid;
+ justify-content: center;
+ align-content: center;
+ gap: #{token(gap)};
+
+ box-sizing: border-box;
+ padding: #{token(padding)};
+ height: #{token(height)};
+ width: #{token(width)};
+ max-width: #{token(max-width)};
+ text-align: center;
+ background-color: #{token(background)};
+ cursor: pointer;
+
+ border: #{token(border-width)} #{token(border-style)} #{token(border-color)};
+}
+
+@mixin base-disabled() {
+ opacity: 0.38;
```

Comment:
Let's make this a `disabled-opacity` token. See other components for a reference to the color emphasis tokens.